app/src/main/res/layout/activity_main.xml

```diff
@@ -26,7 +26,8 @@
             android:fontFamily="@font/poppins_semibold"
             android:lines="2"
             android:maxLines="99"
-            android:text="The average rental \n price around you"
+            android:text="The average rental\nprice around you"
+            android:textAlignment="center"
```

Comment:
This was not justification as I referred to earlier, it's actually just a property called alignment